integration-docs/signing-methods.md

```diff
@@ -1,33 +1,80 @@
-### Implementing Different Signing Methods
+## Signing a transaction
 
-Ideally, a client library should support multiple signing methods. Currently, as a barebone basic demonstration `omg-js` supports transaction signing via raw privatekey and single UTXO as input through `sendSingleTransaction` function. Although, it works for demonstration purpose, a practical usage would require support from signing for users with multiple types of wallets
+The OMG Network childchain uses `eth_signTypedData` as described in [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md)
 
-### Implementing your own sendTransaction function
+This means that a web3 provider (e.g. Metamask) can do the signing.
 
-processing transaction to the childchain consists of 4 major components:
+### Creating and signing with `eth_signTypedData`
+The steps involved in creating and signing a transaction are as follows:
+1. Create the transaction body by specifying inputs, outputs and optional metadata.
+2. Convert the transaction body into `typedData`format.
+3. Call `eth_signTypedData` to sign it. Note that you must `JSON.stringify()` the typed data.
```

Comment:
so `eth_signTypeData` requires an unlocked account on Ethereum node ? Or in this case wallet provider like Metamask ?